terraform/dev/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "aws_region" {
```

Comment:
The `variables.tf` configs declare variables that will be referenced in a respective `main.tf` config, along with some default values.